CONTRIBUTING.md

```diff
@@ -97,28 +97,21 @@ validates all configurations options when provided before generating
 its config file.
 
 The `PrometheusCharm` object interacts with its scrape targets using a
-[charm library](lib/charms/prometheus_k8s/v1/prometheus.py). Using this
-library requires that Prometheus informs its "Consumers" (scrape targets)
-of the actual Prometheus version that was deployed. In order to determine
-this version at runtime `PrometheuCharm` uses the
-[`Prometheus`](src/prometheus_server.py) object. The `Prometheus`
-object provides an interface to a running Prometheus instance. This
-interface is limited to only those aspects of Prometheus required by
-this charm.
+[charm library](lib/charms/prometheus_k8s/v1/prometheus.py).
 
 ### Library Details
```

Comment:
This section should IMO explain what the library is for (enabling providers to have their metrics endpoints scraped by the `prometheus-k8s` charm, but it does not "early enough".